### PR TITLE
TDI-47525 : change thoughtworks version on BOM

### PR DESCRIPTION
**Pull request by clesaec**
https://jira.talendforge.org/browse/TDI-47525 : CVE
**What is the current behavior?** (You can also link to an open issue here)

**What is the new behavior?**

**Please check if the PR fulfills these requirements**

- [ ] The commit message follows Talend standard
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features) ?
- [ ] The code coverage on new code >75%
- [ ] The new code does not introduce new technical issues (sonar / eslint)

**What kind of change does this PR introduce?**

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build / CI related changes
- [ ] Other... Please describe:

**Does this PR introduce a breaking change?**

- [ ] Yes
- [ ] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: components-parent/components-bom/pom.xml [hunk 1]
```diff
@@ -207,7 +207,7 @@
         <osgi.core.version>6.0.0</osgi.core.version>
         <osgi.component.annotations>1.3.0</osgi.component.annotations>
         <talend-proxy.version>1.0.2</talend-proxy.version>
-        <xstream.version>1.4.18</xstream.version>
+        <xstream.version>1.4.19</xstream.version>
```
Comment (ypiel-talend) [inline components-parent/components-bom/pom.xml:210]: I don't well understand the fix.
The CVE: https://sca.analysiscenter.veracode.com/workspaces/WzzF47x/issues/vulnerabilities/100878611
ask to fix with org.apache.jena:jena-core:4.2.0

I have check dependencies on maintenance/7.3, without fix:
> (maintenance/7.3)$ mvn dependency:tree | grep jena
> [INFO] |  +- org.apache.jena:jena-arq:jar:2.13.0:compile
> [INFO] |  +- org.apache.jena:jena-core:jar:2.13.0:compile
> [INFO] |  +- org.apache.jena:jena-iri:jar:1.1.2:compile

And it's the same on the branch clesaec/TDI-47525_CVE that should fix:
> (clesaec/TDI-47525_CVE)$ mvn dependency:tree | grep jena
> [INFO] |  +- org.apache.jena:jena-arq:jar:2.13.0:compile
> [INFO] |  +- org.apache.jena:jena-core:jar:2.13.0:compile
> [INFO] |  +- org.apache.jena:jena-iri:jar:1.1.2:compile

**Review by ypiel-talend — COMMENTED** [review]
Does it fix the CVE ?

File: components-parent/components-bom/pom.xml (hunk 1 above)
Comment (clesaec) [reply components-parent/components-bom/pom.xml:210]: The fix proposed by Vera code is to add 
```xml
<dependency>
 <groupId>com.thoughtworks.xstream</groupId>
 <artifactId>xstream</artifactId>
 <version>1.4.19</version>
</dependency>
```
(dependency graph is about marklogic)

**Comment by jzhao-talend** [comment]
duplicate fix https://github.com/Talend/components/pull/2046
is in qa validation

**Comment by jzhao-talend** [comment]
duplicate fix https://github.com/Talend/components/pull/2046
is in qa validation

**Pull request closed without merging**